### PR TITLE
Create the Mentor Model

### PR DESCRIPTION
**Pull request by pedroruizba4th**

### CODE REVIEW

**Review by cdelacruz8512 — COMMENTED** [review]
Change the Id variable to "MentorId"
Change Type of Interns Variables to public List Interns....
Delete this line [BsonElement("Mentor")] no apply in this class

**Review by cdelacruz8512 — COMMENTED** [review]
Please change the name of the variables like before

**Review by cdelacruz8512 — APPROVED** [review]
the code is correct

File: Internship.API/Models/Mentor.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+{
+    public class Mentor
+    {
+        [BsonId]
+        [BsonRepresentation(BsonType.ObjectId)]
+
+        public int MentorId { get; set; }
+
+        [Required]
+        public string FistName { get; set; }
```
Comment (fasaga) [inline Internship.API/Models/Mentor.cs]: Typo: Should be FirstName

**Review by fasaga — APPROVED** [review]

**Pull request merged**